src/Command/InfectionCommand.php

```diff
@@ -511,8 +427,53 @@ protected function initialize(InputInterface $input, OutputInterface $output)
 
     private function doInitialize(InputInterface $input)
     {
-        $this->io = $this->getApplication()->getIO();
+        $this->consoleOutput = new ConsoleOutput($this->getApplication()->getIO());
         $this->eventDispatcher = $this->getContainer()->get('dispatcher');
         $this->skipCoverage = \strlen(trim($input->getOption('coverage'))) > 0;
     }
+
+    /**
+     * Checks if the infection run passed the constraints like min msi & covered msi.
+     *
+     * @param MetricsCalculator $metricsCalculator
+     *
+     * @return int
+     */
+    private function didRunPassConstraints(MetricsCalculator $metricsCalculator): int
```

Comment:
should it be smth like `hasRunPassConstraints` or `isRunPassConstraints` ?

Reply:
How about `hasRunPassedConstraints` ?